File: abc/465/c/a.c
#include <stdio.h>

int sv(int a, int b){
    float x = b * 2 / 3;
    if (a > x){
        return 1;
    }
    else {
        return 0;
    }
}

int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    if (sv(a, b)){
        printf("Yes");
    }
    else {
        printf("No");
    }
    return 0;
}

File: abc/465/py/a.py

def sv(a, b):
  if a > b * (2/3):
    print('Yes')
  else:
    print('No')
    
def main():
  a, b = map(int, input().split())
  sv(a, b)
  
if __name__ == "__main__":
  main()
  